crates/re_blueprint_tree/src/blueprint_tree.rs

```diff
@@ -546,7 +546,7 @@ impl BlueprintTree {
                             .save_recursive_override_or_clear_if_redundant(
                                 ctx,
                                 &query_result.tree,
-                                &Visible(visible_after),
+                                &Visible(visible_after.into()),
```

Comment:
I think this reads much better:

```suggestion
                                &Visible::from(visible_after),
```

here and everywhere

Reply:
Or 

```suggestion
                                &Visible::new(visible_after),
```

Reply:
We generally don't have `new` methods for components today and I don't want to make this PR _even_ more complex. `From` it is then.